data/json/monsters.json

```diff
@@ -5729,6 +5765,7 @@
     "armor_bash":0,
     "armor_cut":0,
     "vision_day":30,
+    "vision_night":10,
```

Comment:
Random night-murder is literally the whole point.
'complaining that a red @ came out of nowhere at night and stabbed 'em to death as they were walking through the woods'
Walking through the woods in the dark after the cataclysm and expecting to NOT get attacked is the odd point of view.